SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -425,6 +440,9 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_AUTH_SERVER_INTEREST =
 	TArray<Worker_ComponentId>{ // RPCs from clients
 								CLIENT_ENDPOINT_COMPONENT_ID,
 
+								// Position, required for Strategy Worker
+								POSITION_COMPONENT_ID,
```

Comment:
Without this, we can't use Spatial position with grid LB for local entities.

Reply:
Why do we need the Spatial position though? It is a copy of the Unreal actor position updated at a lower frequency.

Reply:
It's in anticipation, as you've said: https://github.com/spatialos/UnrealGDK/pull/2925#discussion_r582115682